enhancements/authentication/automated-service-ca-rotation.md

```diff
@@ -102,29 +102,80 @@ and the functionality could be separately extracted for operator reuse.
     original expiry of the pre-rotation CA to refresh without breaking trust.
 
 - The duration of the service CA should be extended from its current value of
-  12 months to a value that exceeds the maximum expected upgrade interval
-  (currently 12 months). A value of 14 months should be sufficient.
-  - Timelines:
-    - With 12 month CA duration:
+  12 months to 26 months and the minimum CA duration should be extended to 13
+  months. These values ensure that pods will be guaranteed to be restarted in
+  a cluster supported by Red Hat before the expiry of the pre-rotation CA.
+  - The timing of upgrades is key to determining CA duration:
```

Comment:
Add a note here that says that all pods in the openshift control plane automatically reload certificates, keys, and ca-bundles without a pod restart. I've received questions.

Reply:
Done